auth-web/public/config/configuration.json

```diff
@@ -13,7 +13,7 @@
   "AFFIDAVIT_FILE_SIZE": "73KB",
   "PAYMENT_PAYEE_NAME": "BC Registries and Online Services",
   "SENTRY_DSN": "",
-  "SENTRY_ENABLE": "false",
+  "SENTRY_ENABLE": "False",
```

Comment:
I'd recommend converting to lowercase and then comparing.

Reply:
Otherwise if someone switches from False -> true, it might not work the way they'd expect.